setup.md

```diff
@@ -0,0 +1,130 @@
+iOS devices.
+
+    <pre>
+    $ brew tap flutter/flutter
+    $ brew install ideviceinstaller ios-deploy
+    </pre>
+
+## Android setup ##
+
+To develop Flutter apps for Android, you can use either a Mac or a Linux (64-bit) machine:
+
+1. Install [Android Studio](https//developer.android.com/sdk/index.html).
+2. From Android Studio, install the latest Android SDK and Android SDK Platform-Tools, as described
+[here](https://developer.android.com/sdk/installing/adding-packages.html).
+
+### Set up your Android device
+
+To prepare to run and test your Flutter app on Android, you'll need an Android device running
+Android 4.1 (API level 16) or higher.
+
+1. Enable developer mode on your device by visiting **`Settings > About phone`** and
+   tapping the **`Build number`** property seven times.
+2. Enable **`Android debugging`** in **`Settings > Developer options`**.
+3. Using a USB cable, plug your phone into your computer. If prompted on your
+   device, authorize your computer to access your device.
+4. In the terminal, run the `flutter devices` command to verify that Flutter recognizes your
+connected Android device.
+
+By default, Flutter uses the version of the Android SDK where your `adb` tool is based. If
+you want Flutter to use a different installation of the Android SDK, you must set the
+`ANDROID_HOME` environment variable to that specific installation directory.
+
+## Get the Flutter SDK ##
+
+To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
+
+<pre>
+$ git clone https://github.com/flutter/flutter.git -b alpha
+$ export PATH=`pwd`/flutter/bin:$PATH
+</pre>
+
+To complete the setup and to validate that you’ve set everything up correctly, run the following
+command:
+
+```
+$ flutter doctor
+```
+
+The first time your run the `flutter` command, it will download its dependencies and compile
+itself. Subsequent runs should be much faster.
+
+## Atom editor ##
+
+We recommend using [Atom](https://atom.io/) for editing, running, and debugging Flutter apps.
+However, using our command-line tools, you can use any editor to develop Flutter applications.
+
+Download Atom from [atom.io](https://atom.io/) and then install the
+[Flutter package](https://atom.io/packages/flutter). If you need help installing
+packages, consult the [Atom documentation](https://atom.io/docs/v1.3.2/using-atom-atom-packages)
+or [email us](flutter-dev@googlegroups.com).
+
+### Configure Atom for Flutter development ###
+
+Before creating your Flutter project in Atom, you’ll need to perform these pre-flight tasks:
+
+1. In the terminal window, run the `atom` command to launch the editor.
+  ```
+  $ atom
+  ```
+2. Open **Packages > Flutter > Package Settings**, and set the **FLUTTER_ROOT** field to the
+root directory where you installed the Flutter SDK.
+3. Open **Packages > Dart > Package Settings**, and set the **Dart SDK Location** field to the
+`/bin/cache/dart-sdk` directory in your Flutter SDK’s root folder.
```

Comment:
`.../bin/cache/dart-sdk` or `bin/cache/dart-sdk`. With a leading slash it's an absolute path from the root.